### PR TITLE
Update DR docs to cover Solr issue mitigation

### PR DESCRIPTION
**Pull request by joshualieb**
Updates [Site Disaster Recovery](https://pantheon.io/docs/disaster-recovery) Solr and DR section to improve process description and add a reindexing overview for Drupal 7/8 and WordPress. 

## Effect
PR includes the following changes:
- Edited 'Solr and Disaster Recovery' section to include:
  - Better description of what happens and how to fix it
  - Added Solr reindexing overview for D7, D8, and WP.
- Updates & clarifies Solr for D7 doc.
- Fixes formatting issue for Solr for WP doc 

## Remaining Work
- [x] Technical review
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by carolyn-shannon** [comment]
Need SME for WordPress process.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5506 | Difference |
| --- | --- | --- | --- |
| Performance Score | `60` | `49` | `-11` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5506` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18687/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18687/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/disaster-recovery.md [hunk 1]
```diff
@@ -45,20 +46,60 @@ You can also [connect to your Redis instance](/redis/#use-the-redis-command-line
...
+
+![ApacheSolr Indexing](../images/d7-solr-reindex.png)
+
+**Search API Solr module:** Navigate to your Search Index list page at `admin/config/search/search_api` and click the index you need to rebuild. On the index view page, you can either queue all items for reindexing or clear your existing index and re-index in batches.
+
+
+#### WordPress
+
+You can initiate the reindexing process from the WordPress dashboard or via Terminus.
+1. To reindex from the WordPress Dashboard, navigate to `/wp-admin/admin.php?page=solr-power#top#solr_action` and click **Start Index**.
```
Comment (rachelwhitton) [inline source/content/disaster-recovery.md]: Might need a new line before this ordered list - it's rendering as a paragraph in the preview link

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5506 | Difference |
| --- | --- | --- | --- |
| Performance Score | `49` | `47` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5506` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18702/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18702/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**